web/rest/user/features/provider/voicemail/getVoiceMail.feature

```diff
@@ -0,0 +1,50 @@
+Feature: Retrieve voicemails
+  In order to manage voicemails
+  As a user
+  I need to be able to retrieve them through the API.
+
+  @createSchema @userApiContext
+  Scenario: Retrieve a certain voicemail message json
+    Given I add User Authorization header
+     When I add "Accept" header equal to "application/json"
+      And I send a "GET" request to "/voicemails/1"
+     Then the response status code should be 200
+      And the response should be in JSON
+      And the header "Content-Type" should be equal to "application/json; charset=utf-8"
+      And the JSON should be equal to:
+      """
+      {
+           "enabled": true,
+           "name": "Voicemail For User1",
+           "email": "alice@democompany.com",
+           "sendMail": true,
+           "attachSound": true,
+           "id": 1,
+           "user": {
+               "name": "Alice",
+               "lastname": "Allison",
+               "email": "alice@democompany.com",
+               "pass": "*****",
+               "doNotDisturb": false,
+               "isBoss": false,
+               "active": true,
+               "maxCalls": 1,
+               "externalIpCalls": "0",
+               "rejectCallMethod": "rfc",
+               "multiContact": true,
+               "gsQRCode": false,
+               "id": 1,
+               "bossAssistant": null,
+               "extension": null,
+               "timezone": 145,
+               "voicemail": null
```

Comment:
https://github.com/irontec/ivozprovider/blob/6e3f4c0a9c20c1242e81fa32e0455c43814afcab/library/Ivoz/Provider/Domain/Model/User/UserDtoAbstract.php#L222

Yes, it should be removed from the Dto

https://github.com/irontec/ivozprovider/blob/6e3f4c0a9c20c1242e81fa32e0455c43814afcab/library/Ivoz/Provider/Domain/Model/User/UserDtoAbstract.php#L211

That field may be also an issue since its *callACLId* in schema (capitalization doesn't match).

Reply:
https://github.com/irontec/ivozprovider/blob/6e3f4c0a9c20c1242e81fa32e0455c43814afcab/library/Ivoz/Provider/Domain/Model/User/UserDtoAbstract.php#L222

Should be removed from this file. Apparently it has no use.

Reply:
https://github.com/irontec/ivozprovider/blob/6c822c0b37efd3bf18be955bba356a8c9c45bb8d/library/Ivoz/Provider/Infrastructure/Persistence/Doctrine/Mapping/Voicemail.VoicemailAbstract.orm.xml#L41-L45

Voicemails has a 1:1 relation with Users. Users.voicemail is a virtual column populated by doctrine on runtime:
https://github.com/irontec/ivozprovider/blob/6c822c0b37efd3bf18be955bba356a8c9c45bb8d/library/Ivoz/Provider/Infrastructure/Persistence/Doctrine/Mapping/User.User.orm.xml#L25

Reply:
Oooohkey